src/octopeer-github/main/ElementEventBinding/ClickElementEventBinding.ts

```diff
@@ -1,13 +1,11 @@
+/// <reference path="../ElementSelectionBehaviour/ElementSelectionBehaviour.ts"/>
 /**
  * Created by Youri on 04/05/2016.
- */
-
-/**
  * Class for adding click events.
  */
 class ClickElementEventBinding implements ElementEventBinding {
-    private eventType: string = "click";
-    private eventID: EventID  = new EventID(1);
+    public eventType: string = "click";
+    public eventID: EventID  = new EventID(101);
```

Comment:
Sounds good. Since that document is share-able (view and comment only), it might be an idea to put the link in the PR message for "outsiders" (TAs, teachers, basically anyone who is not in our Drive folder).